app/assets/stylesheets/general/_form.scss

```diff
@@ -312,3 +316,75 @@ input[type="file"] {
   font-size: $font-size-small;
   color: $text-color-dark;
 }
+
+.form-tags-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  padding: 0 5px 5px 5px;
+}
+
+.form-tags-input {
+  @extend .inline-input;
+  width: initial;
+  margin-top: 5px;
+  padding: 4px 5px;
+}
+
+.form-tags-tag {
+  display: flex;
+  white-space: nowrap;
+  border-radius: 2px;
+  margin-right: 5px;
+  margin-top: 5px;
+  padding: 4px 2px 3px 4px;
```

Comment:
This is a better example for my previous comment. `4 2 3 4`, Why were those values chosen? Something like `.25rem, .15rem, .2rem, .25rem` makes more sense in my mind, but even then you could probably go with just `.25rem, .15rem` without losing much value (I could be wrong here since I'm not sure what it looks like).

Reply:
I'm not entirely sure why, but the 1 pixel helps center things better. For the sake of not making things look arbitrary though, `.25rem, .15rem` doesn't lose much aesthetic value.